features/iotserver-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/store/extensions/assets/deviceType/themes/store/helpers/asset.js

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+var resources = function (page, meta) {
+    return {
+        js: ['jquery.rating.pack.js', 'async.min.js', 'asset-core.js', 'asset.js','porthole.min.js', 'device-details.js'],
+        css: ['jquery.rating.css', 'asset.css'],
+        code: ['store.asset.hbs']
+    };
+};
+
+var format = function (context) {
+    //adding enriched context for paginating template
+    var log = new Log();
+    var avg = context.asset.rating.average;
+   
```

Comment:
We have dropped ES extended approach and moved code into [[1]](https://github.com/wso2/carbon-device-mgt-plugins/tree/master/components/iot-plugins). This file is does not exist in latest repo Hence cannot fix.